features/desktop/notifications.feature

```diff
@@ -67,6 +68,55 @@ Feature: Notifications
     Then I should see "commented on your post"
     And I should have 1 email delivery
 
+  Scenario: unconnected user comments in reply to my comment to her post
+    Given "alice@alice.alice" has a public post with text "check this out!"
+    When I sign in as "carol@carol.carol"
+    And I am on "alice@alice.alice"'s page
+    And I focus the comment field
+    And I fill in the following:
+        | text        | great post!    |
+    And I press "Comment"
+    Then I should see "less than a minute ago" within ".comment"
+    And I sign out
+    And I sign in as "alice@alice.alice"
+    And I am on "alice@alice.alice"'s page
+    And I focus the comment field
+    And I fill in the following:
+        | text        | great post!    |
+    And I press "Comment"
+    Then I should see "less than a minute ago" within ".comment"
+    And I sign out
+    And I sign in as "carol@carol.carol"
+    And I follow "Notifications" in the header
+    Then the notification dropdown should be visible
+    Then I should see "also commented on"
+    And I should have 2 email delivery
+
+  Scenario: connected user comments in reply to my comment to an uncommented user post
+    Given "alice@alice.alice" has a public post with text "check this out!"
+    And a user with email "bob@bob.bob" is connected with "carol@carol.carol"
+    When I sign in as "carol@carol.carol"
+    And I am on "alice@alice.alice"'s page
+    And I focus the comment field
+    And I fill in the following:
+        | text        | great post!    |
+    And I press "Comment"
+    Then I should see "less than a minute ago" within ".comment"
+    And I sign out
+    And I sign in as "bob@bob.bob"
+    And I am on "alice@alice.alice"'s page
+    And I focus the comment field
+    And I fill in the following:
+        | text        | great post!    |
+    And I press "Comment"
+    Then I should see "less than a minute ago" within ".comment"
+    And I sign out
+    And I sign in as "carol@carol.carol"
+    And I follow "Notifications" in the header
+    Then the notification dropdown should be visible
+    Then I should see "also commented on"
+    And I should have 3 email delivery
+
```

Comment:
Aren't we missing one where Carol comments, Bob comments, Carol comments, Bob should have a notification?